### PR TITLE
Git secrets

### PR DESCRIPTION
**Pull request by hanars**
Because the seqr repo is in the broadinstitute github organization, code is automatically scanned to detect any secrets that were committed. However, it is substantially better to never push up committed secrets in the fist place. This adds a pre-push git hook that will be automatically run whenever you push code to the repo.

Once this is merged, you need to run `git config core.hooksPath .githooks` in the seqr directory to enable the hook. Broad apops come pre-installed with git secrets so you shouldn't need to install anything

### CODE REVIEW

**Comment by hanars** [comment]
passing codacy: https://app.codacy.com/gh/macarthur-lab/seqr/pullRequest?prid=6441773

File: .githooks/pre-push [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  done
+}
+
+function updater {
+  if [ "$1" != "" ]; then cd "$1" > /dev/null; fi
+  printf "%b\n" "${HIGHLIGHT}Scanning ${PWD}${NORMAL}"
+  scan *
+}
+
+updater
```
Comment (ShifaSZ) [inline .githooks/pre-push:38]: Add an extra line?

**Comment by ShifaSZ** [comment]
> Because the seqr repo is in the broadinstitute github organization, code is automatically scanned to detect any secrets that were committed. However, it is substantially better to never push up committed secrets in the fist place. This adds a pre-push git hook that will be automatically run whenever you push code to the repo.
> 
> Once this is merged, you need to run `git config core.hooksPath .githooks` in the seqr directory to enable the hook. Broad apops come pre-installed with git secrets so you shouldn't need to install anything

Should the instruction of `run git config core.hooksPath .githooks` be added to the readme.MD of the project?

**Review by ShifaSZ — APPROVED** [review]
There is a minor issue. It can be merged after updated.

**Comment by hanars** [comment]
> readme

You would think, but our README actually contains no instructions for developing seqrm, and this is only relevant to developers who are pushing code. If we ever wrote good onboarding docs, it would belong in them

**Pull request merged**